src/components/AVPlayer/AVPlayer.js

```diff
@@ -206,7 +206,7 @@ class AVPlayer extends Component {
     const { item } = this.props;
     if (!isEqual(this.state.item, item)) {
       if (this.isUnitExistAndPlaying()) {
-        this.props.chronicles.append('player-stop', this.buildAppendData(this.props.item, this.state.src, this.props.media));
+        this.props.chronicles.append('player-stop', this.buildAppendData(this.props.item, this.state.src, this.props));
```

Comment:
Please look three lines above: there is `item` already. Please take `src` from `this.state` too. Why do you need the whole `this.props`? To take something from it? Or take it here, or take items when already inside.

Reply:
Good point, taking all params now from this.